client/components/CheckIn.js

```diff
@@ -50,7 +58,10 @@ class Checkout extends Component {
     render() {
 
         return (
-            <button type="button" className="btn btn-success" disabled={this.state.lock}> Check in </button>
+            <div>
+                <button type="button" className="btn btn-success" disabled={this.state.lock} onClick = {this.handleClick}>Check In</button>
```

Comment:
I think this is a bootstrap class- will be removed with PWA setup. making note to remove this in next branch